docs/change-log.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 ## v.1.7.0
 - Add tests for phone battery features
+- Fill NA with 0 for the selected event features: (1) each sensor's feature extraction script (2) data cleaning script
```

Comment:
Replace NA with 0 for selected event-based features. Done in each feature extraction script and the data cleaning module